terraform/k8s_ansible.sh

```diff
@@ -0,0 +1,13 @@
+#!/bin/bash
+
+sudo sed -i 's/nova.clouds.archive.ubuntu.com/ubuntu.mirror.ac.za/g' /etc/apt/sources.list
+sudo apt update && sudo apt install -y git python3-venv
+chmod 600 /home/ubuntu/ilifu.pem
+source ~/ilifu-kubernetes-ansible/app-cred.sh
+source ~/ansible/bin/activate
+cd ~; python3 -m venv ansible
```

Comment:
You're sourcing the virtualenv before creating it? is some conditional logic needed here?

Reply:
Thanks, good spot. I'm working on an already live VM so I missed this. Should have done a full test. Fixing it now.